test/src/test/java/org/corfudb/runtime/clients/NettyCommTest.java

```diff
@@ -30,6 +16,17 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
+import javax.annotation.Nonnull;
+import java.io.File;
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import java.util.Collections;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Following Google checkstyle guide - https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines
Keep static imports at the top - https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

Reply:
Done